state/event_table.go

```diff
@@ -338,7 +338,11 @@ func (t *EventTable) LatestEventNIDInRooms(txn *sqlx.Tx, roomIDs []string, highe
 	return
 }
 
-func (t *EventTable) Redact(txn *sqlx.Tx, roomVer string, eventIDs []string) error {
+func (t *EventTable) Redact(txn *sqlx.Tx, roomVer string, redacteeEventIDToRedactEvent map[string]*Event) error {
+	eventIDs := make([]string, 0, len(redacteeEventIDToRedactEvent))
+	for e := range redacteeEventIDToRedactEvent {
+		eventIDs = append(eventIDs, e)
+	}
```

Comment:
One day we might have https://pkg.go.dev/golang.org/x/exp/maps#Values in the stdlib for this